src/catalogue/category/catalogueCard.component.tsx

```diff
@@ -73,17 +74,32 @@ function CatalogueCard(props: CatalogueCardProps) {
         }}
       >
         <CardActions>
-          <Checkbox
-            onClick={(event) => {
-              event.preventDefault();
-              handleCheckboxClick();
-            }}
-            checked={isSelected}
-            inputProps={{
-              'aria-label': 'controlled',
-            }}
-            aria-label={`${catalogueCategory.name} checkbox`}
-          />
+          <Tooltip 
+          title = "Toggle Select"
+          slotProps={{
+            popper: {
+              modifiers: [
+                {
+                  name: 'offset',
+                  options: {
+                    offset: [0, -10],
+                  },
+                },
+              ],
+            },
```

Comment:
Just a question, why was this needed? I can't see much difference between them.

Reply:
All of that code was to access the offset and change the position of the tooltips. It wasn't strictly needed, but I tried to move the tooltips to not block other text.

I realise my laptop's display is smaller, so the offset was noticeable on my end, but not others.

Reply:
Maybe its just me but I cant make them overlap without them

![image](https://github.com/user-attachments/assets/236f4f43-612e-46c1-b1d9-170916807fc4)

Do you have an example you could screenshot?

Otherwise I would probably just remove the offsets - the main thing is that it doesn't cover the component its attached to otherwise the user can just move the mouse again. It saves us having to maintain individual offsets across the site.

Reply:
w/o the offset
![image](https://github.com/user-attachments/assets/0ca4a0e5-53d8-452c-8d74-92b6c9b9ca93)

w/ the offset
![image](https://github.com/user-attachments/assets/bf40b5f5-a75a-41f7-ae7c-6282d2db355e)

However, It was never blocking its own component, so in that case it should be fine to remove the offsets.

Reply:
Yeah I would just remove it and leave it to the tooltip component to decide where to place it then. It was good thought though.